### ISSUE TITLE
Visual Appearance of Control Does Not Match Function

### ISSUE THREAD

**Issue by davidstenger** [uswds/uswds#1644]
Issues:
The Section Title coded as buttons, but are styled to look like a link. Visually sighted users see the links, but for screen reader users it is announced as buttons. Also, there is no differentiation in design between the  Search Title link and Search Title Button. 
See attachment [LinkVSButtons_ExtendedHeader.png](https://drive.google.com/open?id=0ByQeU_iOqP_ia0JkS0dfVVRLazA)

Solution:
Ideally, since the user is navigating to the submenu and not submitting data, the buttons should be coded as links. Provide an underline or additional focus indicators on focus and hover to provide “reassuring” visual confirmation that the element is a link.

The second option will be to create these links as buttons, add the button look style by adding button outline.

**Comment by shawnbot** [comment]
I like the solution of styling links so that they're more visibly distinguished as such. There are lots of pitfalls in casting links as buttons (using `role="button"` or not), such as this one from [the MDN article on the button role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_button_role):

> Be careful when marking up links with the button role. Buttons are expected to be triggered using the Space key, while links are expected to be triggered using the Enter key. In other words, when links are used to behave like buttons, adding `role="button"` alone is not sufficient. It will also be necessary to add a key event handler that listens for the Space key in order to be consistent with native buttons.

Not to mention validation issues of `<a>` without (or with an invalid) `href`.

If the use of `<button>` causes issues because the navigation or header is wrapped in a `<form>`, then I'm pretty sure that you can disable the submit behavior by adding `type="button"`.

**Comment by maya** [comment]
The third item in the screenshot is a link and the rest are buttons. The buttons have a down arrow as a visual affordance to indicate it's an interactive button and are programmatically coded as expandable buttons, whereas the links are coded plainly as links (and without the down arrow).
